charts/registry/templates/registry/registry-deployment.yaml

```diff
@@ -21,23 +21,23 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "chart.fullname" . }}
+  name: {{ include "dtr.fullname" . }}
   labels:
-    {{- include "chart.labels" . | nindent 4 }}
+    {{- include "dtr.labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.registry.replicaCount }}
   selector:
     matchLabels:
-      {{- include "chart.selectorLabels" . | nindent 6 }}
+      {{- include "dtr.selectorLabels" . | nindent 6 }}
   template:
     metadata:
       labels:
-        {{- include "chart.selectorLabels" . | nindent 8 }}
+        {{- include "dtr.selectorLabels" . | nindent 8 }}
     spec:
       securityContext:
         runAsUser: 100
       containers:
-        - name: {{ .Chart.Name }}
+        - name: {{ include "dtr.fullname" . }}
```

Comment:
Whats the reason why you changed it from `{{ .Chart.Name }}` to the fullname?
Chart name was already fine :)

Reply:
Ow, no reason. accidentally replaced. :)
i changed it.